### PR TITLE
docs: docs quickstart user guide

### PR DESCRIPTION
**Pull request by Slug-Boi**
Resolves [#62](https://github.com/orgs/NovoNordisk-OpenSource/projects/2/views/3?pane=issue&itemId=58365024)

### CODE REVIEW

**Review by zqueamish — APPROVED** [review]

**Review by Alex-itu — APPROVED** [review]

**Review by JonasSkjodt — APPROVED** [review]
nicely made

File: docs/user_docs/quickstart.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
+# Welcome to Tech_Radar quickstart guide
+In this guide you will find the essential commands for the program. The program is split into 3 primary commands: Fetcher, Merger, and generator
```
Comment (Haruyzal) [inline docs/user_docs/quickstart.md:2]: Consistency. <3
```suggestion
In this guide you will find the essential commands for the program. The program is split into 3 primary commands: Fetcher, Merger, and Generator.
```

File: docs/user_docs/quickstart.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+Legend for this guide:  
+When running commands you will see <...> anything inside these are required arguments whilst anything inside [...] means optional extra arguments
```
Comment (Haruyzal) [inline docs/user_docs/quickstart.md:5]: Formatting? 🥺 
```suggestion
## Legend for this guide
When running commands, you will see...
**<...>** : Arrows indicate required arguments.
**[...]** : Square-brackets indicate optional arguments.
```

**Review by Haruyzal — COMMENTED** [review]
Amazing work b! Thank you for this great summarization and gifs!

Got a few suggestions 🥺

**Review by August-Brandt — APPROVED** [review]
Looks good

**Pull request merged**